### PR TITLE
update kubectl ref contrib guide

### PR DESCRIPTION
**Pull request by kbhawkey**
See issue #14111 .

### CODE REVIEW

**Comment by kbhawkey** [comment]
Preview:
https://deploy-preview-15199--kubernetes-io-master-staging.netlify.com/docs/contribute/generate-ref-docs/kubectl/

File: content/en/docs/contribute/generate-ref-docs/kubectl.md [hunk 1]
```diff
@@ -48,64 +48,80 @@ information, see
...
 
-Determine the base directory of your clone of the
-[kubernetes/website](https://github.com/kubernetes/website) repository.
-For example, if you followed the preceding step to get the repository, your
-base directory is `$GOPATH/src/github.com/kubernetes/website.`
-The remaining steps refer to your base directory as `<web-base>`.
+Get a clone of the kubernetes/kubernetes repository as k8s.io/kubernetes:
+
+```shell
+git clone https://github.com/kubernetes/kubernetes $GOPATH/src/k8s.io/kubernetes
```
Comment (ialidzhikov) [inline content/en/docs/contribute/generate-ref-docs/kubectl.md]: k/kubernetes is now switched to go modules. Is it still required to clone into $GOPATH?

Comment (kbhawkey) [reply content/en/docs/contribute/generate-ref-docs/kubectl.md]: hello @ialidzhikov . Right, what do you suggest? There are various ways to setup the build for the kubectl reference. I would suggest the most straightforward and repeatable process.

Comment (ialidzhikov) [reply content/en/docs/contribute/generate-ref-docs/kubectl.md]: Starting in Go 1.13 (1 August, 2019), module mode will be the default for all development. The major feature of go modules is that they allow you to decouple from the GOPATH and develop outside of it. 
If possible, I would not git clone in GOPATH.

Comment (kbhawkey) [reply content/en/docs/contribute/generate-ref-docs/kubectl.md]: OK. Is this the best doc which explains the migration to [go modules](https://github.com/golang/go/wiki/Modules)? Typically, the docs reference build is required for what I would call major releases, 1.13, 1.14 with the main k8s build components being kubectl and now kustomize. These two components are or will be in repos outside of the main k8s/k8s repo?

**Pull request merged**